Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -347,29 +363,37 @@ IEnumerator Request(string api_path)
                     if (OnError != null && !IsNetworkError(request)) {
                         OnError(request.downloadHandler.text);
                     }
-                }
-                else if (OnRequestComplete != null) {
+                } else if (OnRequestComplete != null) {
                     OnRequestComplete(request.downloadHandler.text);
                 }
             }
         }
-
+        private void PrepareTusHeaders(UnityWebRequest r, string apiVersion = "3.4")
+        {
+            r.chunkedTransfer = false;
+            r.method = "POST";
+            r.SetRequestHeader("Authorization", "bearer " + token);
+            r.SetRequestHeader("Content-Type", "application/json");
+            r.SetRequestHeader("Accept", "application/vnd.vimeo.*+json;version=" + apiVersion);
```

Comment:
DRY this up by calling `PrepareHeaders` here.

Reply:
🌵 - https://github.com/vimeo/vimeo-unity-sdk/pull/46/commits/2427d90deb58cbaf846728cff0a46aec75b7e04b